tests/hil/tests/rpc/test.c

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2024 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <golioth/client.h>
+#include <golioth/golioth_debug.h>
+#include <golioth/golioth_sys.h>
+#include <golioth/rpc.h>
+#include <golioth/zcbor_utils.h>
+
+LOG_TAG_DEFINE(test_rpc);
+
+static enum golioth_rpc_status on_no_response(zcbor_state_t *request_params_array,
+                                              zcbor_state_t *response_detail_map,
+                                              void *callback_arg)
+{
+    size_t arg_count = 0;
+    while (!zcbor_list_or_map_end(request_params_array))
+    {
+        arg_count++;
+        int err = zcbor_any_skip(request_params_array, NULL);
+        if (!err)
```

Comment:
This is a bit hard to understand since [zcbor doesn't document the return value of the function](https://github.com/NordicSemiconductor/zcbor/blob/75fa5b2703f921433eee32475deed0e9790c968b/include/zcbor_decode.h#L266-L276). Does it return true if the skip was successful?

Reply:
Yeah, this should be probably be `bool ok` instead of `int err`, as in other places where we call zcbor APIs. Will update.